### ISSUE TITLE
translateClassPaths and translateSourcepaths should have same syntax

### ISSUE THREAD

**Issue by confile** [j2objc-contrib/j2objc-gradle#229]
Current state: 

```
  translateClassPaths "lib/json-20140107.jar", 
  translateSourcepaths "${projectDir}/lib/dagger-2.0.1-sources.jar"
```

preferred syntax: 

```
  translateClassPaths "lib/json-20140107.jar", 
  translateSourcepaths "lib/dagger-2.0.1-sources.jar"
```

**Comment by brunobowden** [comment]
Good point. I think we should allow both paths but encourage relative paths, so you can do something like this:

     translateXxxxPaths '../relativeToProject/lib/some.jar'
     translateXxxxPaths "$projectDir}/dir/lib/some.jar"

@advayDev1 - your comments?

**Comment by advayDev1** [comment]
we should just be consistent, if the rule is "all paths configured in j2objc are relative to project.file(X)" sgtm

**Comment by brunobowden** [comment]
I think we should support an absolute path when it might be needed... but perhaps not document that and make everything else relative to the projectDir as you said.

**Comment by advayDev1** [comment]
project.file(X) supports absolute as long as you put the root / in there.
as in, if we document that all paths are resolved per project.file(), all is good

**Comment by brunobowden** [comment]
Fixed by #234
